mono/btls/btls-util.c

```diff
@@ -27,12 +27,11 @@ mono_btls_util_asn1_time_to_ticks (ASN1_TIME *time)
 	ASN1_GENERALIZEDTIME *gtime;
 	struct tm tm;
 	int64_t epoch;
-	int ret;
 	
 	memset (&tm, 0, sizeof (tm));
 
 	gtime = ASN1_TIME_to_generalizedtime (time, NULL);
-	ret = asn1_generalizedtime_to_tm (&tm, gtime);
+	asn1_generalizedtime_to_tm (&tm, gtime);
```

Comment:
Yeah, we could either check in managed code or fallback to some default value in native code.

`asn1_generalizedtime_to_tm()` return 1 on success and 0 on error.  Could you please check the return value and make `mono_btls_util_asn1_time_to_ticks()` return 0 on error?